### DISCUSSION TITLE
Ansible:Parameter store Usage: Expense project

### DISCUSSION THREAD

**Discussion by gkalluru77** [learndevopsonline/learndevopsonline#1593] [Q&A]
Hi,

Getting below error while running the frontend scripts after setting all aws parameter

Workstation IP : 23.22.227.170
Frontend IP : 3.88.4.69

TASK [frontend : copying expense config file] ***********************************************************************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'env' is undefined. 'env' is undefined. 'env' is undefined. 'env' is undefined
fatal: [frontend-dev.gdevops.online]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'env' is undefined. 'env' is undefined. 'env' is undefined. 'env' is undefined"}

PLAY RECAP **********************************************************************************************************************************************************************************
frontend-dev.gdevops.online : ok=2    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

23.22.227.170 | 172.31.43.205 | t3.small | https://github.com/gkalluru77/expense-ansible.git
[ centos@workstation ~/expense-ansible ]$

**Comment by sreedharm07** [comment]
Hi @gkalluru77 
you are not declaring env variable 
execute this command it will works
**command** =   ansible-playbook -i frontend-dev.gdevops.online, expense.yml -e role_name=frontend -e ansible_username=centos -e ansible_password=DevOps321 **-e env=dev**

you forgot to give env variable in command

**Reply by gkalluru77** [reply]
Yeah, Got it. since passing many parameters through command line, overlooked and missed it.  :)
Thank you Sreedhar for quick response.

**Comment by gkalluru77** [comment]
Working fine hence closing it.
